File: app/src/main/java/com/fractalwrench/crazycats/network/responses/ImageWrapperResponse.java
package com.fractalwrench.crazycats.network.responses;

import java.util.List;

public class ImageWrapperResponse {

    private ImageResponse source;
    private List<ImageResponse> resolutions;

    public ImageResponse getSource() {
        return source;
    }

    public void setSource(ImageResponse source) {
        this.source = source;
    }

    public List<ImageResponse> getResolutions() {
        return resolutions;
    }

    public void setResolutions(List<ImageResponse> resolutions) {
        this.resolutions = resolutions;
    }
}

File: app/src/main/java/com/fractalwrench/crazycats/network/responses/ImageWrapperResponse.kt
package com.fractalwrench.crazycats.network.responses

data class ImageWrapperResponse(val source: ImageResponse?, val resolutions: List<ImageResponse>?)
data class ImageResponse(val url: String?, val width: Int, val height: Int)
data class Preview(val images: List<ImageWrapperResponse>?)